SofaKernel/modules/SofaBaseCollision/src/SofaBaseCollision/CubeModel.cpp

```diff
@@ -184,12 +176,7 @@ void CubeCollisionModel::draw(const core::visual::VisualParams* vparams)
         ++level;
         color *= 0.8f;
     }
-    Vec<4,float> c;
-    if (isSimulated())
-        c=Vec<4,float>(1.0f, 1.0f, 1.0f, color);
```

Comment:
if it is like TriangleModel, I think simulated is "gold" and not is white

Reply:
I was not sure but I went for the lazy way :p